docker/dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ubuntu:18.04
+
+COPY *.deb /
+
+RUN apt update && \
+    apt install -y /*.deb && \
+    rm -rf /*.deb /var/lib/apt/lists/*
```

Comment:
A couple of changes in this code block.
1. First, `apt` is not the same as `apt-get`. The `apt-get` command is intended for scripts and the command-line interface will not be changed, whereas `apt` is meant for humans and should not be used by scripts.
2. Second, we are missing `apt-get clean` to remove cache files before committing this layer.
3. Third, the POSIX standard requires that text files end with a newline `\n` character.

Since these docker images are meant for public consumption, instead of internal consumption, we have to be picky about these types of things and maintain compatibility with common open standards like POSIX. So, this code block...
```dockerfile
RUN apt update && \
    apt install -y /*.deb && \
    rm -rf /*.deb /var/lib/apt/lists/*
```
...would become:
```dockerfile
RUN apt-get update && \
    apt-get install -y /*.deb && \
    apt-get clean && \
    rm -rf /var/lib/apt/lists/*

```

Reply:
Done. I'll keep in mind